packages/wallet-sdk/src/CoinbaseWalletProvider.test.ts

```diff
@@ -31,9 +31,7 @@ describe('CoinbaseWalletProvider', () => {
   describe('Request Handling', () => {
     test('handles request correctly', async () => {
       const response1 = await provider.request({ method: 'eth_chainId' });
-      const response2 = await provider.request({ method: 'eth_accounts' });
       expect(response1).toBe(1);
-      expect(response2).toEqual([]);
```

Comment:
this test was wrong. when it's not connected, `eth_accounts` should throw error.